pkg/otelcollector/metricsprocessor/processor.go

```diff
@@ -321,15 +334,15 @@ func (p *metricsProcessor) updateMetricsForFluxMeters(
 	metricValue, found := otelcollector.GetFloat64(attributes, fluxMeter.GetAttributeKey(), treatAsMissing)
 
 	// Add attribute found label to the flux meter metric
-	if found {
-		labels[metrics.ValidLabel] = metrics.ValidTrue
-	} else {
-		labels[metrics.ValidLabel] = metrics.ValidFalse
+	if !found {
+		fluxMeter.DeleteFromHistogram(labels)
```

Comment:
@jmichalak-fluxninja,

I do not understand this logic about deleting a metric in the OTEL processor?